vars/main-6.4.yml

```diff
@@ -116,7 +116,7 @@ auditbeat_defaults:
   output:
```

Comment:
Don't set this to "true", it assumes a direct Elasticsearch connection. It can be overridden in a custom config by the user.